inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -456,122 +430,130 @@ population_data_age_months <- gen_population(total_pop = 5000,
 #   strata = NULL) %>%
 #   rename(age_category = groups,
 #     population = n)
+
+
+## estimate population size by region proportion
+population_data_region <- gen_population(total_pop = 5000,         # set the total population
+  groups = c("Village A", "Village B", "Village C", "Village D"),  # set the groups
+  proportions = c(0.221, 0.174, 0.355, 0.245),                     # set the proportions for each group
+  strata = NULL) %>%               # do not stratify by gender
+  rename(patient_origin = groups,  # rename columns (syntax is NEW NAME = OLD NAME)
+         population = n)
+
 ```
 
 
-```{r lab_data}
-## OPTIONAL: this section can be used to read in lab data
-## make sure that your unique identifiers (e.g. case_number) match with the linelist! 
-## - [ ] DELETE THE FAKE DATA SECTION!!
 
-## Read data ------------------------------------
-## When you copy and paste file paths, you need to change \ to /.
-##
 
-## Excel file
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+lab_data
+
+If you have lab data, you can read it in here.
+You will need to make sure that your identifier (e.g. case_number) matches the
+linelist.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+```{r lab_data}
+## Read data -------------------------------------------------------------------
+## This step reads in your lab data from Excel.
+## You may need to rename your columns.
+
 ## to read in a specific sheet use "which"
 # lab_data <- rio::import(here("labresults.xlsx"), which = "Sheet1")
 
-## repeat same cleaning steps as in standardise_clean_data code chunk as appropriate
+## repeat preparation steps as appropriate
 ## make sure your unique ID variable name matches!
 
-## Fake data ------------------------------------ 
-## THE BELOW CAN BE DELETED IF YOU HAVE YOUR OWN (REAL) LABORATORY DATA! 
-## its just to be able to demonstrate script in this template
-
-# # generate artificial lab tests results
-# lab_results <- linelist_cleaned %>% 
-#                 select(case_number) %>% 
-#                 mutate(test_result = sample(c("Positive", "Negative"),
-#                                             nrow(linelist_cleaned), 
-#                                             replace = TRUE)
-#                        )
-# 
-# # merging linelist with lab dataset 
+## Merge lab to linelist -------------------------------------------------------
 # linelist_cleaned <- left_join(linelist_cleaned, lab_results, 
 #                               by = "case_number")
 ```
 
 
 
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+browse_data
```

Comment:
ùncomment all the code and set to eval = FALSE - then give explanation here that you dont want this section running in your markdown - but is just to have code for browsing.

Reply:
as above re: eval = FALSE